pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -185,6 +224,61 @@ func newLibvirtDomainManager(connection cli.Connection, virtShareDir string, not
 	return &manager, nil
 }
 
+func detectEFIROMS(arch, ovmfPath string) *efiConfiguration {
```

Comment:
This factory function can go to the new package as well.

Reply:
Done